packages/discovery-provider/src/utils/auth_middleware.py

```diff
@@ -87,7 +91,16 @@ def wrapper(*args, **kwargs):
                         logger.info(
                             f"auth_middleware.py | authed_user_id: {authed_user_id}"
                         )
-            return func(*args, **kwargs, authed_user_id=authed_user_id)
+            return (
```

Comment:
I learned today that if you pass a kv argument that isn't consumed, it will often generate an error.
So I had to do this weirdness of conditionally calling the inner function differently. I would have preferred to conditionally add to the arguments list. But I could not wrap my head around arguments in python well enough to do it.

Reply:
I think you could include in it kwargs, like this

```
def inner(*args, **kwargs):
  print("fun!", args, kwargs)

def outer(*args, **kwargs):
  kwargs["thing"] = True
  fun(*args, **kwargs)
```

Reply:
Oh interesting. Maybe we should do that for the other variable as well then?
In my reading I couldn't find any suggestion that you can modify kwargs like that. But I'll give it a try.

Reply:
[updated](https://github.com/AudiusProject/audius-protocol/pull/8238/commits/14e5ed148ba62124c6561f946ace96d6dfbd6df6#diff-3a03921c05b5547cafa0e18268303afa5ac49ab9679e307499bd6c40d531f886R94)